at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -365,9 +386,18 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
         ..limit = _atClient.getPreferences()!.syncPageLimit
         ..isPaginated = true;
       _logger.finer('** syncBuilder ${syncBuilder.buildCommand()}');
-      var syncResponseJson = JsonUtils.decodeJson(DefaultResponseParser()
-          .parse(await _remoteSecondary.executeVerb(syncBuilder))
-          .response);
+      List syncResponseJson;
+      try {
+        syncResponseJson = JsonUtils.decodeJson(DefaultResponseParser()
+            .parse(await _remoteSecondary.executeVerb(syncBuilder))
+            .response);
+      } on AtException catch (e) {
+        e.stack(AtChainedException(Intent.syncData,
+            ExceptionScenario.remoteVerbExecutionFailed, e.message));
+        _logger.severe(
+            'Exception occurred in fetching sync response : ${e.getTraceMessage()}');
+        rethrow;
```

Comment:
i think we should not rethrow here.. log and continue syncing rest of the response from server

Reply:
Ok, Will remove the rethrow